### PR TITLE
adds microprofile helidon and open liberty project

### PR DESCRIPTION
**Pull request by otaviojava**
* https://github.com/platformsh/template-microprofile-helidon/pull/1
* https://github.com/platformsh/template-microprofile-openliberty/pull/1

### CODE REVIEW

File: templates/microprofile-helidon/files/src/main/java/sh/platform/template/HelloWorldResource.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import javax.ws.rs.Produces;
+
+@Path("")
+@RequestScoped
+public class HelloWorldResource {
+
+    @GET
+    @Produces("text/plain")
+    public String doGet() {
+        return "hello from Platform.sh";
```
Comment (chadwcarlson) [inline templates/microprofile-helidon/files/src/main/java/sh/platform/template/HelloWorldResource.java]:
```suggestion
        return "Hello world - a simple Microprofile Helidon template for Platform.sh";
```

File: templates/microprofile-openliberty/files/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Open Liberty for Platform.sh
+
+This project provides a starter kit for Open Liberty Eclipse MicroProfile projects hosted on Platform.sh.
```
Comment (chadwcarlson) [inline templates/microprofile-openliberty/files/README.md]: This line should also be it's own first <p> tag in .platform.template.yaml

File: templates/microprofile-openliberty/files/src/main/java/sh/platform/template/HelloWorldResource.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import javax.ws.rs.Produces;
+
+@Path("")
+@RequestScoped
+public class HelloWorldResource {
+
+    @GET
+    @Produces("text/plain")
+    public String doGet() {
+        return "hello from Platform.sh";
```
Comment (chadwcarlson) [inline templates/microprofile-openliberty/files/src/main/java/sh/platform/template/HelloWorldResource.java]:
```suggestion
        return Hello world - a simple Microprofile Open Liberty template for Platform.sh;
```

File: templates/microprofile-openliberty/files/README.md (hunk 2 above)
Comment (otaviojava) [reply templates/microprofile-openliberty/files/README.md]: Fixed, thank you @chadwcarlson

**Pull request merged**